### PR TITLE
First draft of BLISS results section

### PR DESCRIPTION
**Pull request by zmbc**
I am unsure about many aspects of this, so I figured it would be good to open it up to review earlier than I normally would. For example, I still have not actually included citations here. So consider this a work in progress; I may ask for re-review, but hoping to get initial reactions at this stage.

If it would be easier, we could also collaborate on this in Word or another format where it is a bit easier to directly edit it together.

### CODE REVIEW

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 1]
```diff
@@ -334,18 +334,143 @@ In the NHANES example, the full first sentence of the above quote was:
...
+If it isn't possible to narrow this down or distinguish what might be important, we can consider
+all gender- and sex-related factors to be variables of interest.
+
+When we present our findings, we have two goals:
+
+* Accurately convey what associations we are looking for, and what the causal structures might be.
+  We should strive to be as precise as possible, and not rely on "common sense"
+  to indicate which gender- and sex-related factors are related to a health outcome.
+  Research shows that if descriptions of sex or gender differences are imprecise,
+  readers tend to ascribe them to sex, due to misconceptions about what gender can affect. [cite]
```
Comment (aflaxman) [inline docs/source/model_design/general_reference_material/bliss/index.rst:350]: got a citation to add here?

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 2]
```diff
@@ -334,18 +334,143 @@ In the NHANES example, the full first sentence of the above quote was:
...
+  of sex- and gender-related factors. [cite guidelines for collecting these]
+
+On the other hand, if we were doing descriptive epidemiology and didn't have even an educated
+guess of what specific sex- and gender-related factors could be associated with our outcome:
+
+.. pull-quote::
+
+  Sex and gender are multi-dimensional concepts.
+  Collectively, they span a number of biological (sex-related) and sociocultural
+  (gender-related) factors which are highly correlated and causally entangled with one other. [citations!]
```
Comment (aflaxman) [inline docs/source/model_design/general_reference_material/bliss/index.rst:414]: I have a few citations I can give you for this if you don't have some already

**Review by aflaxman — APPROVED** [review]

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 3]
```diff
@@ -334,18 +334,143 @@ In the NHANES example, the full first sentence of the above quote was:
...
+  Frequently in health research, differences are attributed wholly to sex-related factors, when
+  in fact gender-related factors also play a part in those differences. [citation]
+  Since prevalence of <outcome> could be influenced by sex- and/or gender-related factors,
+  an ideal analysis would report prevalence by multiple of these factors.
+
```
Comment (SylLutze) [inline docs/source/model_design/general_reference_material/bliss/index.rst:419]: I think having an example somewhere - not sure if here is the right spot - about gender casual pathways might be helpful for readers. Things I think about in CVD: is one gender more likely to go to the doctor sooner, is one gender's pain taken more seriously in the emergency room. These are gender things but its easy to imagine how they would affect outcomes.

**Review by SylLutze — APPROVED** [review]
Overall I like this a lot!

**Review by pletale — APPROVED** [review]
This is fantastic - thanks so much Zeb!!

**Pull request merged**